fieldservice_portal/views/fsm_order_template.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+    <template id="portal_my_fsm_orders" name="My Work Orders">
+        <t t-call="portal.portal_layout">
+            <t t-set="breadcrumbs_searchbar" t-value="True" />
+            <t t-call="portal.portal_searchbar">
+                <t t-set="title">Work Orders</t>
+            </t>
+            <t t-if="not grouped_orders">
+                <p>There are no Work Orders in your account.</p>
+            </t>
+            <t t-else="">
+                <t t-call="portal.portal_table">
+                    <t t-foreach="grouped_orders" t-as="orders">
+                        <thead>
+                            <tr
+                                t-if="groupby != 'none'"
+                                t-attf-class="{{'thead-light'}}"
+                            >
+                                <!--
+                                careful: this does not work if groupby is not a relation
+                                        towards a model that has a 'name' field
+                                -->
+                                <th colspan="6" class="text-left text-muted">
+                                    <em class="font-weight-normal text-muted">
+                                        Work Orders for
+                                        <t
+                                            t-esc="searchbar_groupby[groupby]['label']"
```

Comment:
`t-esc` has been deprecated in 15.0 in favour of `t-out`

Reply:
Done